src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -295,6 +297,11 @@ export const ArtworkAppFragmentContainer = createFragmentContainer(
         ...PricingContext_artwork
       }
     `,
+    me: graphql`
```

Comment:
@damassi Taking further look now, I wonder if I should be using a user from`SystemContextConsumer` instead here? Is this the use-case for it?

Reply:
Yup, `SystemContextConsumer` is the right path here because that's one less request and its already available. 

In React class components the consumer is needed to grab things but inside of function components its even easier: 
```js
import { useSystemContext } from 'Artsy'
...
const { user } = useSystemContet()
```

Reply:
Note that on line 253 there's an intermediate function component that you can use to grab `me` from `useSystemContext`. For all of the Relay components below that need `me` access I would suggest a similar path.

Reply:
So spiked a bit further here. Unfortunately we only have `identityVerified` on `me` at the moment. As such, it's not straightforward to refactor to using the `SystemContextConsumer` (which exposes a ` User`, not a `me`, which are weirdly not the same type...).

Given this work, there are a couple of metaphysics changes I would like to make:

1. Extending `User` with `identityVerified`
2. Extending `User` with `needsIdentityVerificationFor(sale)` to encapsulate and centralize the pretty complex  `userNeedsIdentityVerification` method.

---

For now, I'm thinking about _removing_ the usage of `SystemContextConsumer` in this component considering we have a `me` now via the more rudimentary fragement based approach, and then _just_ use `SystemContextConsumer` (and remove the `me` fragement) after MP is changed per above.

Reply:
Does the injected `user` object consistently have other fields we use in this PR, like `identityVerified`?

Reply:
@yuki24 Does not at the moment, added a screenshot to comment above to clarify

Reply:
Having `identityVerified` on user will be helpful from a system-context perspective in the future. Iterating on this would be good since adding `me` everywhere requires `me` to be passed as props and `__generated__` fragments to be emitted. 

We might need to look closely at the user object in Force before it gets passed into `buildServer/ClientApp` because that's likely where this will be change will need to be made (in addition to the MP changes).